docs/semgrep-appsec-platform/jira.md

```diff
@@ -38,11 +51,19 @@ Once you have enabled the Jira integration, you must complete the following step
 1. Select the **Subdomain** for the Jira instance you want to use.
 2. Select the **Default project** where the Jira tickets will be created.
 3. Select the **Issue type** you want created with your Semgrep findings.
-4. If you'd like to see an example of the content Semgrep populates your Jira ticket with, click **See preview**.
+4. Optional: To see an example of the content Semgrep populates your Jira ticket with, click **See preview**.
 5. Click **Save changes** to proceed.
 
 ![Jira configuration screen](/img/jira-subdomain.png#md-width)
-**_Figure._** The Jira configuration screen.
+_**Figure.** The Jira configuration screen._
+
+### Automatic detection of other Jira projects
+
+The Jira integration automatically detects other Jira projects in your subdomain if those projects have the same **Issue type** as the default project. [When you triage a finding](#code), you can choose which project to create the tickets in.
+
+:::caution Same name, different ID
```

Comment:
We could also specify here that we specifically need the projects and issue types to be company-managed, not team-managed, for this to work https://support.atlassian.com/jira-software-cloud/docs/what-are-team-managed-and-company-managed-projects/

Reply:
Good callout, thank you!